docs/.eleventy.js

```diff
@@ -55,11 +55,23 @@ module.exports = function(eleventyConfig) {
     // Load site-specific data
     const siteName = process.env.ESLINT_SITE_NAME || "en";
 
+    /**
+     * Determines whether the given version is a prerelease.
+     * @returns {boolean} `true` if it is a prerelease, `false` otherwise.
```

Comment:
```suggestion
     * Determines whether we are in the prerelease phase
     * @returns {Promise<boolean>} `true` if there is a prerelease of the next major version, `false` otherwise.
```